### PR TITLE
Default Styles lesson: Add missing word

### PR DESCRIPTION
**Pull request by racodess**
## Because
Under the "What are default styles and where do they come from?" header, the last sentence is missing a word.

## This PR

- Add the missing word "vary" to the sentence.

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: intermediate_html_css/intermediate_css_concepts/default_styles.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ This section contains a general overview of topics that you will learn in this l
 
 ### What are default styles and where do they come from?
 
-As you have worked on projects, you likely observed default styles applied to certain elements, such as larger and bolder headings on `h1` elements, and blue, underlined links on `a' elements. There is also a good chance you struggled with things like default margins and padding. These styles are part of the user-agent stylesheets, ensuring basic styling for webpages without CSS. Each browser has it's own set of user-agent stylesheets so the default styles do slightly between browsers.
+As you have worked on projects, you likely observed default styles applied to certain elements, such as larger and bolder headings on `h1` elements, and blue, underlined links on `a' elements. There is also a good chance you struggled with things like default margins and padding. These styles are part of the user-agent stylesheets, ensuring basic styling for webpages without CSS. Each browser has it's own set of user-agent stylesheets so the default styles do vary slightly between browsers.
```
Comment (MaoShizhong) [inline intermediate_html_css/intermediate_css_concepts/default_styles.md:14]:
```suggestion
As you have worked on projects, you likely observed default styles applied to certain elements, such as larger and bolder headings on `h1` elements, and blue, underlined links on `a' elements. There is also a good chance you struggled with things like default margins and padding. These styles are part of the user-agent stylesheets, ensuring basic styling for webpages without CSS. Each browser has its own set of user-agent stylesheets so the default styles do vary slightly between browsers.
```

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Good catch, thank you! While we're at it, would you mind removing the highlighted unnecessary apostrophe as well?

File: intermediate_html_css/intermediate_css_concepts/default_styles.md (hunk 1 above)
Comment (racodess) [reply intermediate_html_css/intermediate_css_concepts/default_styles.md]: Done!

**Review by MaoShizhong — APPROVED** [review]
Many thanks!

**Pull request merged**